bugbug/models/duplicate.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from itertools import islice
+
+from imblearn.over_sampling import BorderlineSMOTE
+from sklearn.calibration import CalibratedClassifierCV
+from sklearn.compose import ColumnTransformer
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import LabelEncoder
+from sklearn.svm import LinearSVC
+
+from bugbug import bug_features, bugzilla, feature_cleanup
+from bugbug.model import BugCoupleModel
+
+count = 6000
```

Comment:
It isn't a rule, but usually in Python constants are uppercase.
Please also make this a bit more informative, `count` is not a great name.
I would define three different constants, one for duplicate, one for non-duplicate, and one for hybrid.

Reply:
Ideally, we'd need to set a limit only for the third case : (nondup - non-dup). I have set `count` now just for running it on my system. Should I remove it for the first two cases?

Reply:
Are you able to run the model on your system if you set the limit only for the third case?

Reply:
No, it runs out of memory.

Reply:
OK, for now use a limit that allows you to run the model, then we'll make this configurable with options.

Reply:
Filed https://github.com/mozilla/bugbug/issues/521 about this.